syntax = "proto3";

message Products {
   int32 product_id = 1;           // Product ID
  string product_name = 2;        // Product name
  int32 product_price = 3;        // Product price
  string product_sku = 4;         // SKU code
  string product_category = 5;    // Product category
  string last_modified = 6;  
}

message ProductImages {
    string image_url = 1;
    string image_name = 2;
    int32 product_id = 3; 
}
